src/app.js

```diff
@@ -330,5 +302,18 @@ export default function (app) {
 
     return element
   }
+
+  function event(type) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    (subscriptions[type] || []).forEach(function(cb) { cb.apply(null, args) });
+    (subscriptions["*"] || []).forEach(function(cb) { cb.apply(null, type) });
```

Comment:
To be frankly even the whole `event` function could be removed. I just left it there because the `hook` function uses `Array.reduce` which I bet is less performant than `Array.forEach`.